mlflow/__init__.py

```diff
@@ -27,15 +27,35 @@
 
 For a lower level API, see the :py:mod:`mlflow.tracking` module.
 """
-from mlflow.version import VERSION as __version__  # pylint: disable=unused-import
-from mlflow.utils.logging_utils import _configure_mlflow_loggers
-import mlflow.tracking._model_registry.fluent
-import mlflow.tracking.fluent
+import google.protobuf
+from packaging.version import Version
+
+# This is a patch for protobuf <= 3.6.1, to make mlflow compatible with protobuf <= 3.6.1
+if Version(google.protobuf.__version__) <= Version("3.6.1"):
```

Comment:
@WeichenXu123 rather than patching the generated protobuf code, is it possible to change the way we refer to enum values in the implementation of `DatabricksArtifactRepository`?

Reply:
> change the way we refer to enum values in the implementation of DatabricksArtifactRepository

There're many enum instance besides DatabricksArtifactRepository, and each enum contains many values. So changing each of them become big work.

Reply:
> rather than patching the generated protobuf code

Here I patching  `google.protobuf.internal.enum_type_wrapper`, not protobuf generated code.